scoping_document.md

```diff
@@ -112,48 +112,48 @@ The corresponding acceptance criteria supplement the specification of the requir
-| E02.06 | As an app user, I want to be able to see various hotlines for technical, privacy-related, health-related, and psychological questions, as well as for verification of test results, so that I can receive further information or answers to my questions. | 1. The app offers access to a technical hotline and a hotline for obtaining a phone TAN.<hr/>2. The times the hotline is available (such as 24/7) is displayed.<hr/>3. Phone numbers can be called directly from the app. |
+| E02.05 | As an app user, I want to be able to display the terms of use and data privacy information, so that I'm able to understand the usage of my data and so that the app conforms to DSGVO. | 1. The app provides simple access to the terms of use and data privacy information. |
+| E02.06 | As an app user, I want to be able to see hotline numbers for technical, privacy-related, health-related, and psychological questions, as well as for verification of test results, so that I can receive further information or answers to my questions. | 1. The app offers access to a technical hotline and a hotline for obtaining a phone TAN.<hr/>2. The times the hotline is available (such as 24/7) is displayed.<hr/>3. Phone numbers can be called directly from the app. |
 
 ### Use in the Regular Process
 | # of user story ID | User story | Acceptance criteria |
 |-----------------|------------|--------------------|
-| E03.01 | As an app user, I want to be able to activate and deactivate the app, so that I can switch the functions on and off. | 1. The functions for generating pseudonymized IDs and sending them to nearby devices can be activated and deactivated. <hr/>2. The consequences of activation/deactivation are explained. |
-| E03.02 | As an app user, I want to be able to reset to the app to the delivery defaults, so that I can reconfigure it. | 1. The app can be reset to the delivery defaults with a setting option. The saved traces have to be deleted through the system settings. |
-| E03.03 | As an app user, I want to be able to adjust the app settings (access permissions, such as notifications) in a menu, so that I can manage the app functions and accesses. | 1. App users can call a menu for app settings. <hr/>2. Notifications can be activated and deactivated.<hr/>3. Access to the functions for generating pseudonymized IDs and sending them to nearby devices can be activated and deactivated.<hr/>4. Before access permissions are deactivated, I receive information as to which app functions will no longer work (in full). |
+| E03.01 | As an app user, I want to be able to activate and deactivate the app, so that I can switch the functions on and off. | 1. The functions for generating pseudonymized IDs and sending them to nearby devices can be activated and deactivated. <hr/>2. If the app user activated the app, the app explains the consequences of activation. If the app user deactived the app, the app explains the consequences of deactivation. |
```

Comment:
> If the app user activated the app, the app explains the consequences of activation. If the app user deactived the app, the app explains the consequences of deactivation. 

Mixing of past- and present tense, rather:

> If the app user activate**s** the app, the app explains the consequences of activation. If the app user deactive**s** the app, the app explains the consequences of deactivation. |

Reply:
Agree